src/locate/app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+import logging
+import os, sys
+from flask_cors import CORS
+from flask_pymongo import PyMongo
```

Comment:
This line produce an error: `Error: While importing "locate.app", an ImportError was raised:

Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/site-packages/flask/cli.py", line 240, in locate_app
    __import__(module_name)
  File "/Users/danielogenrwot/Documents/Projects/AirQo Platform/AirQo-api/src/locate/app.py", line 5, in <module>
    from flask_pymongo import PyMongo
ModuleNotFoundError: No module named 'flask_pymongo'`

Reply:
Never mind...  It was an issue with my macOS environment setup.